woocommerce/woocommerce
# Add: shipping plugins in shipping task based on location

### All Submissions:






### Changes proposed in this Pull Request:



See https://github.com/woocommerce/woocommerce/issues/36248 for full background.

This PR adds the functionality to feature more shipping partners in the onboarding shipping task.

Closes #36248 .







### How to test the changes in this Pull Request:






1. Set up a new WooCommerce installation
2. During the onboarding wizard, select one of the countries in the list for testing, with a physical product and without additional plugins 
3. On the WooCommerce admin home screen, look for the "Add shipping costs" task and click on it
4. It should bring you to the shipping task page where you can add a shipping partner
5. The third step of the stepper should be a panel with either one or two shipping providers depending on the country you selected earlier
6. The buttons on the panels should successfully install and activate the plugin, except for Envia which has a manual installation method.
7.  A successful installation should occur and bring you back to the homescreen.

Note: The Skydropx provider has an installation issue right now which fails on the first try and succeeds on retry. This will be resolved by either the plugin developers fixing it before this is merged, or otherwise we will revert to the manual installation instructions which does not automatically install the plugin.

https://user-images.githubusercontent.com/27843274/220021752-7613b6e1-c8ae-4071-be03-b80e9cc30158.mp4

### Countries

This is the list of countries and which partners will be featured.
LATAM

    Mexico: Skydropx
    Colombia: Skydropx
    Chile: Envia
    Argentina: Envia
    Peru: Envia
    Brazil: Envia
    Uruguay: Envia
    Guatemala: Envia

Note that in the future, as Skydropx expands geographically, we would also add them to countries where Envia is being promoted. Both would be featured side by side.
Europe

    France: Sendcloud & Packlink
    Spain: Sendcloud & Packlink
    Italy: Sendcloud & Packlink
    Germany: Sendcloud & Packlink
    UK: Sendcloud & ShipStation
    Netherlands: Sendcloud
    Austria: Sendcloud
    Belgium: Sendcloud




### Other information:




### FOR PR REVIEWER ONLY:
